utils/prometheus/README.md

```diff
@@ -0,0 +1,555 @@
+| consensus_best_block_height_number     | IntGauge  |          | best Height of the chain                                        |
+| consensus_target_syn_number            | IntGauge  |          | syning Height target number                                     |
+| consensus_num_txs                      | Gauge     |          | Number of transactions                                          |
+| consensus_node_memory                  | IntGauge  |          | Node's primary memory                                           |
+| consensus_node_cpu                     | IntGauge  |          | Node's cpu load                                                 |
+| consensus_state_cache_size             | IntGauge  |          | used state cache size                             			  |
+| p2p_peers_number                       | IntGauge  |          | Number of peers node's connected to                             |
+| p2p_peer_receive_bytes_per_sec         | IntGauge  |          | number of bytes received from a given peer                      |
+| p2p_peer_send_bytes_per_sec            | IntGauge  |          | number of bytes sent to a given peer                            |
+| Resource_receive_bytes_per_sec(Future) | IntGauge  |          | Operating System of bytes received                              |
+| Resource_send_bytes_per_sec(Future)    | IntGauge  |          | Operating System of bytes sent                                  |
+| Resource_cpu_use(Future)               | IntGauge  |          | Operating System cpu load                                       |
+| Resource_disk_use(Future)              | IntGauge  |          | Operating System disk use                                       |
+| validator_sign_prevote(Future)         | IntGauge  | validator addr | validator sign vote list                               	  |
+| validator_sign_precommit(Future)       | IntGauge  | validator addr | validator sign commit list                                |
+
+
+## Start Prometheus
+### Install prometheus
+
+https://prometheus.io/download/
+```bash
+wget <download URL>
+tar -zxvf <prometheus tar file>
+```
+
+### Edit Prometheus config file
+
+You can visit [prometheus.yml](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus.yml) to download default `prometheus.yml`.
+
+Then edit `prometheus.yml` and add `jobs` :
+
+```yaml
+      - job_name: kusama
+          static_configs:
+          - targets: ['localhost:33333']
+            labels:
+              instance: local-validator
+```
+
+> Note：value of targets is ip:port which used by substrate monitor 
+
+### Start Prometheus
+
+```bash
+cd <prometheus file>
+./prometheus
+```
+
+> The above example, you can save `prometheus.yml` at `~/volumes/prometheus` on your host machine
+
+You can visit `http://localhost:9090` to see prometheus data.
+
+
+
+## Start Grafana
+### Install Grafana
+https://grafana.com/docs/installation/debian/
```

Comment:
Again, I don't think it is worth explaining how to install Grafana nor Prometheus.

Reply:
Agreed

Reply:
ok, Agreed !Milestone 2 will provide a fully configured dashboard, so we will consider removing it in this release and tagging grafana official links the next release.